tuhi/protocol.py

```diff
@@ -650,12 +650,16 @@ class MsgGetName(Msg):
     interaction = Interactions.GET_NAME
     opcode = 0xbb
     protocol = ProtocolVersion.ANY
+    name = ""
```

Comment:
sorry, I only saw this now. This isn't correct, it needs to go into the constructor, otherwise you're using a classmethod here and future instantiations will keep appending to this value.

Reply:
I changed the  code and added a constructor. I'm no python wizard, I hope this is correct. I tried to just pass arguments along to the super constructor while setting the name on `self`.